scripts/upgrade

```diff
@@ -30,13 +30,12 @@ if [[ -z "${secret_key:-}" ]]; then
 fi
 
 #=================================================
-# BACKUP SQLITE DATABASE
+# BACKUP SQLITE DATABASE AND DATAS
 #=================================================
 ynh_script_progression --message="Backup of the DB in temporary file...." --weight=5
 
-if [ -f "$install_dir/data/association.sqlite" ];then
-cp -a $install_dir/data/association.sqlite /tmp/
-fi
+# copy data directory backup
+cp -a $install_dir/data /tmp/data
```

Comment:
That part aint great from a security point of view, this could accidentally allow access to any user of the machine if permissions are not set properly ... Is this really needed considering the data folder is listed in --keep in ynh_setup_source later ?

Though I realize there's a `--full_replace=1` hmf

But still, if you do this you should at least create a proper "secure" temporary directory with `mktemp -d`, not just yolo cp stuff into `/tmp` directly ...

Also theoretically `/var` and `/tmp` could be in different partitions so, not great if there's a huge amount of data to cp ...